Jenkinsfile

```diff
@@ -99,10 +99,6 @@ pipeline {
                 // tasks["Openshift v3.11, oss"] = {
                 //  sh "./bin/start --docker --oss --oc311"
                 // }
```

Comment:
@eladkug @InbalZilberman do we have a plan to run tests of Conjur OSS with OCP? if not, can we remove this to enhance code maintainability?

Reply:
we can remove this. We dont support it (document updated as well in the past)
`## Supported Platforms
- GKE

- Openshift 3.9, 3.10, and 3.11 _*(DAP only)*_`

Reply:
thanks @eladkug .

@izgeri can you please remove these commented out tasks while you're at it?